dissect/cobaltstrike/beacon.py

```diff
@@ -617,7 +617,7 @@ def parse_beacon_gate(data: bytes) -> BeaconGateOptions:
 
 def beacon_gate_options_string(bgo: BeaconGateOptions) -> list[str]:
     """Return the enabled BeaconGate WinAPI's as a list of strings"""
-    options = {k for k, v in bgo._values.items() if v}
```

Comment:
does cstruct provide a better or a public api for accessing the values? If not, that might be a nice contract/feature to add so it doesn't break in a future cstruct update :)

Reply:
Yeah so although `__values__` has a lot of underscores, that is mostly to avoid collisions with user defined fields; it is considered  part of the public interface. 

Alternative would be to loop over the field names and get the attributes, I think this is slightly cleaner